src/components/common/CustomSwitch.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    margin: theme.spacing(2, 0),
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '55%',
+  },
+  label: {
+    display: 'block',
+  },
+}));
+
+const CustomSwitch = ({ switchStatus, switchDispatch, switchLabel }) => {
```

Comment:
What do you think about renaming it to `SwitchWithLabel`? `LabelledSwitch`?

Reply:
Additionally I would rename `switchStatus` to `isChecked`, it sounds more logical for me.

Reply:
And `switchDispatch` to `onToggle`